modules/observable/src/main/scala/notebook/ObsWebSocketService.scala

```diff
@@ -19,19 +19,35 @@ class ObsWebSocketService(system: ActorSystem, val channel: Concurrent.Channel[J
   class LocalActor extends Actor with ActorLogging {
     var remote: ActorRef = null
 
+    var channels:List[Concurrent.Channel[JsValue]] = List(channel)
+
     override def preStart() {
       val remoteDeploy = Await.result(remoteDeployFuture, 2 minutes)
       remote = context.actorOf(Props[ObsServiceRemoteActor].withDeploy(remoteDeploy))
     }
 
     def receive = {
+      case m@("add", channel: Concurrent.Channel[JsValue]) =>
+        //println(s"Adding channel $channel to channels $channels")
+        channels = channel :: channels
+
+      case m@("remove", channel: Concurrent.Channel[JsValue]) =>
+        //println(s"Removing channel $channel to channels $channels")
+        channels = channels.filter(_ != channel)
+        if (channels.isEmpty) {
+          //println(s"Killing WebSocketObservableActor since no more channels are attached")
+          sender() ! akka.actor.PoisonPill
```

Comment:
what does this kill? and is it intentional? 
what if it's a page refresh? what if user will reopen the notebook soon?
can it be related to #531 ?